src/tasks/level9.ts

```diff
@@ -448,7 +448,11 @@ export const ChasmQuest: Quest = {
       completed: () => step("questL09Topping") >= 1,
       do: () => {
         visitUrl(`place.php?whichplace=orc_chasm&action=bridge${get("chasmBridgeProgress")}`); // use existing materials
+        if(have($item`bat wings`) && get("chasmBridgeProgress") >= 25) {
```

Comment:
Could you make a separate task for this? Instead of needed to swap to bat wings every time we turn in items.

Reply:
Absolutely!